### DISCUSSION TITLE
Do "assets" url count as storage to be paid?

### DISCUSSION THREAD

**Discussion by angeousta** [community/community#67816] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

When you paste an image inside an issue, it becomes an url like

https://github.com/OrganizationName/RepositoryName/assets/123456789/abcdef-ghij-qwerty 

do those kind of files count as storage that has to be paid or are they free?

**Comment by SenaJp** [comment, marked as answer]
It's free my friend

**Reply by angeousta** [reply]
Thanks, is there any source on this?

**Reply by SenaJp** [reply]
You will only have a maximum size of the file you want to attach:

https://docs.github.com/en/get-started/writing-on-github/working-with-advanced-formatting/attaching-files

**Reply by SenaJp** [reply]
Hope this helps!

**Reply by angeousta** [reply]
Thanks, it does
